.github/workflows/check.yml

```diff
@@ -36,48 +36,56 @@ jobs:
-    - run: git config --global --add safe.directory /__w/Vulkan-Samples/Vulkan-Samples
-    - run: git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
-    - run: python3 /usr/local/bin/check_copyright_headers.py $TARGET_BRANCH
+      - uses: actions/checkout@v3
+      - run: git config --global --add safe.directory /__w/Vulkan-Samples/Vulkan-Samples
+      - run: git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
+      - run: python3 /usr/local/bin/check_copyright_headers.py $TARGET_BRANCH
 
   snake_case_check:
     name: Snake Case Check
     container: ghcr.io/khronosgroupactions/snake-case-check:latest
     runs-on: ubuntu-latest
     continue-on-error: true
     steps:
-    - uses: actions/checkout@v3
-    - run: python3 /usr/local/bin/snake_case_check.py $TARGET_BRANCH > snake-report.txt
-    - run: if [ $(grep -c '@@' snake-report.txt) -gt 0 ]; then cat snake-report.txt; exit 1; fi
+      - uses: actions/checkout@v3
+      - run: python3 /usr/local/bin/snake_case_check.py $TARGET_BRANCH > snake-report.txt
+      - run: if [ $(grep -c '@@' snake-report.txt) -gt 0 ]; then cat snake-report.txt; exit 1; fi
 
   clang_format:
     name: Clang Format Check
     container: ghcr.io/khronosgroupactions/clang-tools:15.0.0
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - run: git config --global --add safe.directory /__w/Vulkan-Samples/Vulkan-Samples
-    - run: git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
-    - name: Run Clang Format diff
-      id: clang-diff
-      run: echo "changes=$(echo $(git diff -U0 --no-color $TARGET_BRANCH | python3 /usr/share/clang/clang-format-diff.py -p1 -v -sort-include))" >> $GITHUB_OUTPUT
-    - name: Count Diff Lines
-      continue-on-error: true
-      id: count-diff
-      run: echo "line-count=$(echo "${{ steps.clang-diff.outputs.changes }}" | grep -c +++)" >> $GITHUB_OUTPUT
-    - name: Assert
-      run: if test ${{ steps.count-diff.outputs.line-count }} -gt 0; then echo "${{ steps.clang-diff.outputs.changes }}"; exit 1; fi
+      - uses: actions/checkout@v3
+      - run: git config --global --add safe.directory /__w/Vulkan-Samples/Vulkan-Samples
+      - run: git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
+      - name: Run Clang Format diff
+        id: clang-diff
+        run: |
+          echo "changes=$(echo $(git diff -U0 --no-color $TARGET_BRANCH | python3 /usr/share/clang/clang-format-diff.py -p1 -v -sort-include))" >> $GITHUB_OUTPUT
+          echo "$(echo $(git diff -U0 --no-color $TARGET_BRANCH | python3 /usr/share/clang/clang-format-diff.py -p1 -v -sort-include))" >> clang-issues.diff
+      - name: Count Diff Lines
+        continue-on-error: true
+        id: count-diff
+        run: echo "line-count=$(echo "${{ steps.clang-diff.outputs.changes }}" | grep -c +++)" >> $GITHUB_OUTPUT
+      - name: 'Upload Artifact'
+        uses: actions/upload-artifact@v4
+        with:
+          name: clang-issues.diff
+          path: clang-issues.diff
+          retention-days: 3
```

Comment:
As the artifact (`clang-issues.diff`) is generated in the `Run Clang Format diff` step, I would have expected that this step (`Upload Artifact`) is right after that step, not after the `Count Diff Lines` step.
Besides that, why do you have apostrophes around `Upload Artifact`